.changeset/many-bananas-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+"create-fuels": patch
+---
+
+feat: `create fuels` template optimisations
```

Comment:
@danielbate How does this PR relate to the template revamp, and will the changes in this PR be overridden by those from 👇 ?
 - https://github.com/FuelLabs/fuels-ts/pull/3125

Reply:
This PR is improvements to the current templates based on feedback from @SilentCicero / @rishabhkeshan .

#3125 completely changes and overwrites the current templates, but either implements or supersedes the above feedback.

Unfortunately this PR took a little longer than hoped. Hopefully we can get this in for tomorrows release with #3125 making next week.